src/main/scala/org/bitcoins/core/gen/TransactionGenerators.scala

```diff
@@ -208,7 +208,7 @@ trait TransactionGenerators extends BitcoinSLogger {
     (creditingTx,outputIndex) = buildCreditingTransaction(signedScriptSig.redeemScript, amount)
     (signedTx,inputIndex) = buildSpendingTransaction(creditingTx,signedScriptSig, outputIndex, witness)
     signedTxSignatureComponent = WitnessV0TransactionSignatureComponent(signedTx,inputIndex,
-      scriptPubKey, Policy.standardScriptVerifyFlags,amount)
+      scriptPubKey, Policy.standardScriptVerifyFlags,amount, SigVersionWitnessV0)
```

Comment:
Instead of passing the constant `SigVersionWitnessV0` we need to call `BitcoinScriptUtil.parseSigVersion`